### PR TITLE
Added Sentim-API, a free API for sentiment analysis

### PR DESCRIPTION
**Pull request by sharadcodes**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -537,10 +537,10 @@ API | Description | Auth | HTTPS | CORS |
 | [Deepcode](https://www.deepcode.ai/docs/Overview%252FOverview) | AI for code review | No | Yes | Unknown |
 | [Dialogflow](https://dialogflow.com) | Natural Language Processing | `apiKey` | Yes | Unknown |
 | [Keen IO](https://keen.io/) | Data Analytics | `apiKey` | Yes | Unknown |
+| [Sentim-API](https://sentim-api.herokuapp.com) | A free REST API for sentiment analysis | No | Yes | Yes |
 | [Time Door](https://timedoor.io) | A time series analysis API | `apiKey` | Yes | Yes |
 | [Unplugg](https://unplu.gg/test_api.html) | Forecasting API for timeseries data | `apiKey` | Yes | Unknown |
 | [Wit.ai](https://wit.ai/) | Natural Language Processing | `OAuth` | Yes | Unknown |
-
```
Comment (SuperSandro2000) [inline README.md:543]: keep this line please.

**Comment by sharadcodes** [comment]
Fixed Now

**Comment by sharadcodes** [comment]
Can It be merged now ?

File: README.md [hunk 2]
```diff
@@ -537,6 +537,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Deepcode](https://www.deepcode.ai/docs/Overview%252FOverview) | AI for code review | No | Yes | Unknown |
 | [Dialogflow](https://dialogflow.com) | Natural Language Processing | `apiKey` | Yes | Unknown |
 | [Keen IO](https://keen.io/) | Data Analytics | `apiKey` | Yes | Unknown |
+| [Sentim-API](https://sentim-api.herokuapp.com) | A free REST API for sentiment analysis of text | No | Yes | Yes |
```
Comment (yannbertrand) [inline README.md:540]:
```suggestion
| [Sentim-API](https://sentim-api.herokuapp.com) | Text sentiment analysis | No | Yes | Yes |
```

**Comment by yannbertrand** [comment]
Thank you

**Comment by wobsoriano** [comment]
@sharadcodes down already?

**Comment by sharadcodes** [comment]
It's up, dyno hours were exhausted, apologizes for inconvenience

**Pull request merged**